tests/test_validation.py

```diff
@@ -798,3 +814,36 @@ def test_validate_rag_inputs_matching_lists() -> None:
     assert len(result) == list_length
     assert result[0] == "Q: query 1 C: context 1"
     assert result[1] == "Q: query 2 C: context 2"
+
+
+def test_disable_trustworthiness_without_custom_criteria_raises_error(tlm_api_key: str) -> None:
+    """Test that disable_trustworthiness=True without custom_eval_criteria raises ValueError."""
+    with pytest.raises(
+        ValidationError, match="^disable_trustworthiness is only supported when custom_eval_criteria is provided"
+    ):
+        TLM(api_key=tlm_api_key, options=TLMOptions(disable_trustworthiness=True))
```

Comment:
[nit] use the more standard way to pass the arg:

```suggestion
        TLM(api_key=tlm_api_key, options={"disable_trustworthiness": True})
```